cmd/k3sctl/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/datawire/teleproxy/pkg/dtest"
+
+	"github.com/spf13/cobra"
+)
+
+// Version holds the version of the code. This is intended to be overridden at build time.
+var Version = "(unknown version)"
+
+func main() {
+	var k3s = &cobra.Command{
+		Use:           "k3sctl",
+		Short:         "k3sctl",
+		Long:          "k3sctl - manage a local docker registry and kubernetes cluster",
+		SilenceErrors: true,
+		SilenceUsage:  true,
+	}
+
+	var logFile *os.File
+
+	k3s.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
+		logFile, err := os.OpenFile("/tmp/k3sctl.log", os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
```

Comment:
Can we have this obey `$TMPDIR`?  (`os.TempDir()` gives `$TMPDIR` or `/tmp` as appropriate).